aea/channel/local.py

```diff
@@ -228,6 +228,45 @@ def disconnect(self, public_key: str) -> None:
             self.agents.pop(public_key, None)
 
 
+class LocalNodeChannel(Channel):
+    """Channel implementation for the local node."""
+
+    def __init__(self, public_key: str, local_node: LocalNode):
+        """
+        Initialize a OEF proxy for a local OEF Node (that is, :class:`~oef.proxy.OEFLocalProxy.LocalNode`.
+
+        :param public_key: the public key used in the protocols.
+        :param local_node: the Local OEF Node object. This reference must be the same across the agents of interest.
+        """
+        self.public_key = public_key
+        self.local_node = local_node
+
+    def connect(self) -> Optional[Queue]:
+        """
+        Set up the connection.
+
+        :return: A queue or None.
+        """
+        return self.local_node.connect(self.public_key)
+
+    def disconnect(self) -> None:
+        """
+        Tear down the connection.
+
+        :return: None.
+        """
+        return self.local_node.disconnect(self.public_key)
+
+    def send(self, envelope: Envelope) -> None:
+        """
+        Send an envelope.
+
+        :param envelope: the envelope to send.
+        :return: None.
+        """
+        return self.local_node.send(envelope)
```

Comment:
this just wraps the local node methods to ensure consistent interface across channels